tests/compilation_tests/database_test.py

```diff
@@ -13,7 +13,7 @@
 class TestDataBase(unittest.TestCase):
     # DataBase instances to test on
     robot_state_default = Robot_State(xpos=0, ypos=0, heading=0, epsilon=0, max_velocity=0,
-                          radius=0)
+                          radius=0, phase = Phase.SETUP)
```

Comment:
are the params needed if they're the default value?

Reply:
same as above will remove the ones w same value as default o7